### PR TITLE
added send verifcation email button to listings tab for unverified hosts

### PR DESCRIPTION
**Pull request by syd-tan**
# Limited features of unverified hosts

Closes [#166](https://github.com/staynomad/Nomad-Back/issues/166) on Nomad-Back 

# Summary of change

- Unverified hosts no longer have access to creating listings under the listings tab on the My Account page until they become verified.
- Instead, hosts will be prompted to verify their account, which they can do by clicking the Send Verification Email button.

# Testing/Verification

- Sign in to an unverified host account.
- The listings tab should look like this:
Screen Shot 2021-08-03 at 10 07 10 PM
- Click the send verification button, and an email should be sent to the email linked with the account.
Screen Shot 2021-08-03 at 10 07 40 PM
 - Click on the link provided in the email, which should direct to a verification page.
 - The user should now be verified, and the listings tab should allow access to creating and viewing listings. 
Screen Shot 2021-08-03 at 10 07 59 PM

### CODE REVIEW

**Comment by syd-tan** [comment]
Also, one thing I noticed was that clicking on the Activate Account button on the email didn't lead me to the correct page. Can someone double check if that happens too? We might need to make a ticket for it.

**Review by aidenszeto — APPROVED** [review]
Yeah it looks like the button is redirecting to /MyAccount rather than the verification link. @syd-tan could you make a ticket for updating it?

Everything else looks good to me though!

File: src/components/matches/listing/listings.component.js [hunk 1]
```diff
@@ -166,154 +167,174 @@ class Listings extends Component {
...
-                onChange={() =>
-                  this.setState((prevState) => ({
-                    filters: {
-                      ...prevState.filters,
-                      showDrafts: !this.state.filters.showDrafts,
-                    },
-                  }))
-                }
+    const user = this.props.userInfo;
+    if (user.isVerified === false) {
```
Comment (aidenszeto) [inline src/components/matches/listing/listings.component.js:171]: Could we change this to `if (!user.isVerified)` instead? There are some instances where `isVerified`is not a field for the user so we should just check for any falsy values

**Review by aidenszeto — CHANGES_REQUESTED** [review]

File: src/components/matches/listing/listings.component.js (hunk 1 above)
Comment (syd-tan) [reply src/components/matches/listing/listings.component.js:171]: fixed!

**Review by aidenszeto — APPROVED** [review]

**Review by nishantbalaji — APPROVED** [review]
lgtm

**Pull request merged**